src/dmd/semantic3.d

```diff
@@ -1000,6 +1000,28 @@ private extern(C++) final class Semantic3Visitor : Visitor
                 if (global.params.useOut == CHECKENABLE.off)
                     fens = null;
             }
+
+            // if the function body or the in contracts contain "assert(__ctfe)" set the compile time only flag
+            {
```

Comment:
Why this scope?

Reply:
I might want to move this check at some point.
The Scope helps me find it